docs/source/testing/scenarios.rst

```diff
@@ -3,16 +3,135 @@
+restapi: true
+
+* **CEPH SOURCE**: (string) indicate whether a ``dev`` or ``stable`` release is
+  needed. A ``stable`` release will use the latest stable release of Ceph,
+  a ``dev`` will use ``shaman`` (http://shaman.ceph.com)
+
+* **SUBNETS**: These are used for configuring the network availability of each
+  server that will be booted as well as being used as configuration for
+  ceph-ansible (and eventually ceph). The two values that are **required**::
+
+    public_subnet: 192.168.13
+    cluster_subnet: 192.168.14
+
+* **MEMORY**: Memory requirements (in megabytes) for each server, e.g.
+  ``memory: 512``
+
+* **interfaces**: some vagrant boxes (and linux distros) set specific
+  interfaces. For Ubuntu releases older than Xenial it was common to have
+  ``eth1``, for CentOS and some Xenial boxes ``enp0s8`` is used. **However**
+  the public Vagrant boxes normalize the interface to ``eth1`` for all boxes,
+  making it easier to configure them with Ansible later.
+
+.. warning:: Do *not* change the interface from ``eth1`` unless absolutely
+             certain that is needed for a box. Some tests that depend on that
+             naming will fail.
+
+* **disks**: The disks that will be created for each machine, for most
+  environments ``/dev/sd*`` style of disks will work, like: ``[ '/dev/sda', '/dev/sdb' ]``
+
+* **vagrant_box**: We have published our own boxes to normalize what we test
+  against. These boxes are published in Atlas
+  (https://atlas.hashicorp.com/ceph/). Currently valid values are:
+  ``ceph/ubuntu-xenial``, and ``ceph/centos7``
+
+The following aren't usually changed/enabled for tests, since they don't have
+an impact, however they are documented here for general knowledge in case they
+are needed:
+
+* **ssh_private_key_path** : The path to the ``id_rsa`` (or other private SSH
+  key) that should be used to connect to these boxes.
+
+* **vagrant_sync_dir** : what should be "synced" (made available on the new
+  servers) from the host.
+
+* **vagrant_disable_synced_folder** : (bool) when disabled, it will make
+  booting machines faster because no files need to be synced over.
+
+* **os_tuning_params** : These are passed onto ceph-ansible as part of the
+  variables for "system tunning". These shouldn't be changed.
+
 
 .. _scenario_wiring:
 
 Scenario Wiring
 ---------------
+Scenarios are just meant to provide the Ceph environment for testing, but they
+do need to be defined in the ``tox.ini`` so that they are available to the test
+framework. To see a list of available scenarios, the following command (ran
+from the root of the project) will list them, shortened for brevity::
+
+    $ tox -l
+    ...
+    jewel-ansible2.2-centos7_cluster
+    ...
+
+ These scenarios are made from different variables, in the above command there
+ are 3:
+
+* jewel: the Ceph version to test
+* ansible2.2: the Ansible version to install
+* ``centos7_cluster``: the name of the scenario *and* directory
```

Comment:
scenario names don't always match up with the directory name and it's not necessarily required